src/utils/compose.js

```diff
@@ -0,0 +1,3 @@
+export default function(...funcs) {
+  return Component => funcs.reduce((comp, func) => func(comp), Component);
+}
```

Comment:
Jeg synes det var litt vanskelig å henge med på denne funksjonen her. Jeg skjønner at denne sammen med `withState` sørger for at vi kan holde på formdataen uten å lagre det i den redux-storen. Men kanskje vi kan prøve å skrive det litt om sammen slik at det er lettere å forstå hva som skjer?